connector/oidc/oidc.go

```diff
@@ -262,15 +262,28 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 	if !found {
 		return identity, fmt.Errorf("missing \"%s\" claim", userNameKey)
 	}
+
+	emailInScopes := func() bool {
+		for _, s := range c.oauth2Config.Scopes {
+			if s == "email" {
+				return true
+			}
+		}
+		return false
+	}()
+
 	email, found := claims["email"].(string)
 	if !found {
-		return identity, errors.New("missing \"email\" claim")
+		if emailInScopes {
```

Comment:
You could merge these two `if`s. The default value of email is an empty string because of the type.

Reply:
Yep, you are right. Fixed.